src/main/java/org/wildfly/security/keystore/LdapKeyStore.java

```diff
@@ -102,7 +103,7 @@ public LdapKeyStore build() {
                     filterAlias, filterCertificate, filterIterate, createPath, createRdn, createAttributes, aliasAttribute,
                     certificateAttribute, certificateType, certificateChainAttribute, certificateChainEncoding,
                     keyAttribute, keyType);
-            return new LdapKeyStore(spi, null, null);
+            return new LdapKeyStore(spi, new WildFlyElytronProvider(), "LdapRealm");
```

Comment:
I'm aware. My point is that we cannot instantiate new dummy Provider in LdapKeyStore like:
new LdapKeyStore(spi, new Provider("WildFlyElytron", 0.0, "WildFly Elytron Dummy Provider"), "LdapRealm");